doc/leak_tracking/DETECT.md

```diff
@@ -24,39 +54,32 @@ import 'package:leak_tracker/leak_tracker.dart';
 ...
 
 enableLeakTracking();
-MemoryAllocations.instance
+FlutterMemoryAllocations.instance
       .addListener((ObjectEvent event) => dispatchObjectEvent(event.toMap()));
 runApp(...
 
 ```
 
 3. Run the application in debug mode and watch for a leak related warnings.
-If you see a warning, open the link to investigate the leaks.
 
-TODO(polina-c): implement the link and add example of the warning.
+TODO(polina-c): add example of the warning.
 
-### Flutter tests
+## Limitations
 
-Wrap your tests with `withLeakTracking` to setup automated leak verification:
+### By environment
 
-```dart
-test('...', () async {
-  final leaks = await withLeakTracking(
-    () async {
-      ...
-    },
-  );
-
-  expect(leaks, leakFree);
-});
-```
+Leak tracker is enabled for unit tests in Flutter packages.
```

Comment:
At this time, leak tracking is supported only for unit tests within Flutter packages.